core/htmlmaker/bandaid.js

```diff
@@ -1,74 +0,0 @@
-var fs = require('fs');
-var cheerio = require('cheerio');
-var file = process.argv[2];
-
-fs.readFile(file, function processTemplates (err, contents) {
-  $ = cheerio.load(contents, {
-          xmlMode: true
-        });
-
-  //function to replace element, keeping innerHtml & attributes
-  function replaceEl (selector, newTag) {
-    selector.each(function(){
-      var myAttr = $(this).attr();
-      var myHtml = $(this).html();
-      $(this).replaceWith(function(){
-          return $(newTag).html(myHtml).attr(myAttr);
-      });
-    });
-  }
-
-  // paragraphs to remove after conversion
-  $('blockquote + p.SpaceBreak-Internalint, aside + p.SpaceBreak-Internalint, pre + p.SpaceBreak-Internalint').remove();
-  $('blockquote + p.BookmakerProcessingInstructionbpi, aside + p.BookmakerProcessingInstructionbpi, pre + p.BookmakerProcessingInstructionbpi').remove();
```

Comment:
Just want to confirm: did you do some tests with the new htmlmaker js to make sure lines 22 and 23 here will no longer be necessary?